src/strategies/erc721-with-tokenid-list-weighted/examples.json

```diff
@@ -0,0 +1,23 @@
+[
+  {
+    "name": "Example query",
+    "strategy": {
+      "name": "erc721-with-tokenid-list-weighted",
+      "params": {
+        "address": "0x30cDAc3871c41a63767247C8D1a2dE59f5714e78",
+        "symbol": "Reaper(s)",
+        "tokenIdWeightLists": [
+          { "weight": 1, "ids": ["2112", "2151", "2871"] },
+          { "weight": 10, "ids": ["3221", "3587", "4179"] }
+        ]
```

Comment:
Hi @paste I think there should limit for `tokenIdWeightLists` and `ids` else, people may pass huge arrays and there could be memory issues on the server.